bot/utils/converters.py

```diff
@@ -0,0 +1,16 @@
+import discord
+from discord.ext.commands.converter import MessageConverter
+
+
+class BetterMessageConverter(MessageConverter):
+    """A converter that handles embed-suppressed links like <http://example.com>."""
+
+    async def convert(self, ctx: discord.ext.commands.Context, argument: str) -> discord.Message:
+        """Wrap the commands.MessageConverter to handle <> delimited message links."""
+        # It's possible to wrap a message in [<>] as well, and it's supported because its easy
+        if argument.startswith("[") and argument.endswith("]"):
```

Comment:
Is there any reason to have square brackets around links? As far as I know, there is no reason to do that.

Reply:
It's similar to <>, in that it suppresses the embed, but unlike <> it remains visible in the message content.

Reply:
It doesn't seem to suppress the embed for me. I might be doing something wrong?

![image](https://user-images.githubusercontent.com/52584399/94555428-944e3480-0229-11eb-9887-04b8f2469282.png)

Reply:
Use <> instead of []

Reply:
I'm honestly not sure what is up with that. I was under the impression it worked on _all_ embeds, but apparently it doesn't. The testing I've done suggests that <> works for everything, but [] only seems to fail when applied to message links. 

I can take the check for [] out if desired, it was pretty opportunistic (there's space, so why not support it) although it looks like it's either bugged, or just doesn't work atm.

Reply:
How odd. I think it's fine to leave it in then.